### ISSUE TITLE
persistent group sites

### ISSUE THREAD

**Issue by hsl1** [stefanXO/Tab-Manager-Plus#141]
chrome have group sites by colour wich is fine however they will go away when we close chrome,
can you make them presistent when we close like group Orange has 3 sites   red have 2 sites etc

and with the color dot in your plugin presentation

**Comment by cookwjc** [comment]
Agreed - would like to see support for chrome's native tab groups inside a single window in TMP as well as the ability to make them persistent.

**Comment by isalew** [comment]
Duplicate of #16
